app/src/main/java/org/asteroidos/sync/MainActivity.java

```diff
@@ -378,4 +369,29 @@ public void onConfigurationChanged(Configuration newConfig) {
         overridePendingTransition(0, 0);
         startActivity(getIntent());
     }
+
+    static private class SynchronizationHandler extends Handler {
+        private final MainActivity mActivity;
+
+        SynchronizationHandler(MainActivity activity) {
+            mActivity = activity;
+        }
+
+        @Override
+        public void handleMessage(Message msg) {
+            switch (msg.what) {
+                case SynchronizationService.MSG_SET_LOCAL_NAME:
+                    mActivity.handleSetLocalName((String) msg.obj);
+                    break;
+                case SynchronizationService.MSG_SET_STATUS:
+                    mActivity.handleSetStatus(msg.arg1);
+                    break;
+                case SynchronizationService.MSG_SET_BATTERY_PERCENTAGE:
+                    mActivity.handleBatteryPercentage(msg.arg1);
```

Comment:
For consistency with handleSetLocalName and handleSetStatus this should probably be handleSetBatteryPercentage

Reply:
bumping this